### ISSUE TITLE
Can't use Go 1.11 module

### ISSUE THREAD

**Issue by sriio** [hidevopsio/hiboot#59]
**Describe the bug**
As hiboot use a fork of Viper, the imports (github.com/hidevopsio/viper) does not correspond to the module name (module github.com/spf13/viper in go.mod) 

```
go: hidevops.io/viper@v1.3.1: parsing go.mod: unexpected module path "github.com/spf13/viper"
go: error loading module requirements
```

**Expected behavior**
Use directly the repo github.com/spf13/viper or change the module name of github.com/hidevopsio/viper

**Steps to reproduce the bug**
```
cd $GOPATH/src/github.com/hidevopsio/
git clone https://github.com/hidevopsio/hiboot-web-app-demo
cd hiboot-web-app-demo
GO111MODULE=on go mod init github.com/hidevopsio/hiboot-web-app-demo
GO111MODULE=on go build -v
```

**Version**
V0.11.1

**Environment**
Archlinux
Go 1.11.2

**Comment by sriio** [comment]
Temporary workaround for me: add this line to my go.mod

```
replace hidevops.io/viper => github.com/sriio/viper v1.3.4
```

With that line and the right versions of dependencies, it's okay.

**Comment by john-deng** [comment]
@sriio This issue is solved on v1.0.0

**Comment by sriio** [comment]
That works perfectly, thank you ! :+1:
